_episodes/12-cmdline.md

```diff
@@ -666,25 +666,25 @@ the program now does everything we set out to do.
 
 > ## Arithmetic on the Command Line
 >
-> Write a command-line program that does addition and subtraction:
+> Write a command-line program that does addition and subtraction (you could also try multiplication and division operators):
```

Comment:
Since the solution contains multiplication and division, I suggest we include it along with addition and subtraction in the main challenge description. Also, since the arithmetic only works with 2 numbers, we should probably specify that as well.
```suggestion
> Write a command-line program that performs addition, subtraction,
 multiplication, or division with two numbers:
```

Reply:
I agree with Lauren's suggestion to improve this change and I'd like to propose further improvement:
```suggestion
> Write a Python program that adds, subtracts, multiplies, or divides two numbers provided on the command line:
```

Reply:
I like the version by @maxim-belkin .